pkg/capabilities/oci/manifest/manifest.go

```diff
@@ -0,0 +1,55 @@
+package manifest
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/kubewarden/policy-sdk-go/constants"
+	cap "github.com/kubewarden/policy-sdk-go/pkg/capabilities"
+	specs "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+type concreteImageManifestResponse struct {
+	image *specs.Manifest
+	index *specs.Index
+}
+
+func (r concreteImageManifestResponse) ImageManigest() *specs.Manifest {
+	return r.image
+}
+
+func (r concreteImageManifestResponse) IndexManifest() *specs.Index {
+	return r.index
+}
+
+// GetOCIManifestDigest computes the digest of the OCI object referenced by image
```

Comment:
the godoc string needs to be updated, this was copied from GetOCIManifestDigest()

Reply:
Fixed